src/vs/workbench/contrib/terminalContrib/suggest/common/terminalSuggestConfiguration.ts

```diff
@@ -39,6 +40,17 @@ export const terminalSuggestConfiguration: IStringDictionary<IConfigurationPrope
 		default: false,
 		tags: ['experimental'],
 	},
+	[TerminalSuggestSettingId.Providers]: {
+		restricted: true,
+		markdownDescription: localize('suggest.providers', "Controls which providers are enabled for terminal suggestions.  Also be aware of the {0}-setting which controls if extensions are able to provide suggestions.", `\`#${TerminalSuggestSettingId.EnableExtensionCompletions}#\``),
+		type: 'object',
+		properties: {},
+		default: {
+			'terminal-suggest': false,
+			'pwsh-shell-integration': false,
```

Comment:
We should set one of them as the default. We could set pwsh as true and flip the default later, or switch to the extension but we'll need to make sure the change is announced in release notes in that case since it's a soft breaking change.

Reply:
I had forgotten that since this is behind the other setting, it's safe to enable by default. I think `terminal-suggest` should be true by default, so will do that.